docs/en/contributor/Building-Alluxio-From-Source.md

```diff
@@ -144,17 +144,6 @@ To build against Cloudera's releases, just use a version like `$apacheRelease-cd
 -Phadoop-2 -Dhadoop.version=2.0.0-cdh4.7.0
 ```
 
-#### MapR
```

Comment:
@apc999 @ns1123  actually, are we only removing support for MapR underfs modules? or are we removing support for MapR distribution of Hadoop as well? right now I removed both docs.  we might have to add this back in, if we want to continue to support MapR release?

Reply:
Since MapR isn't even supporting their distribution of MapR, I don't think we will be either :)
Jokes aside, we should be okay removing all reference of MapR from the docs. Older doc versions will have the information if people really need it.